Makefile.am

```diff
@@ -32,6 +32,10 @@ OSX_PLIST=$(top_builddir)/share/qt/Info.plist #not installed
 OSX_QT_TRANSLATIONS = da,de,es,hu,ru,uk,zh_CN,zh_TW
 
 DIST_DOCS = $(wildcard doc/*.md) $(wildcard doc/release-notes/*.md)
+DIST_CONTRIB = $(wildcard $(top_srcdir)/contrib/*.bash-completion) \
+	       $(top_srcdir)/contrib/debian \
```

Comment:
There is still a `bitcoin-qt.desktop` file and a `bitcoin-qt.protocol` file in there (makes it open bitcoin-qt when clicking a bitcoin: url in kde apps like kmail for example) but I could live without it... Problem is still apparently everything in `debian/` is GPL2 according to the copyright file in that folder:

> Files: debian/*
> Copyright: 2010-2011, Jonas Smedegaard <dr@jones.dk>
>            2011, Matt Corallo <matt@bluematt.me>
> License: GPL-2+

So I guess it makes sense to remove it for now, until someone has a better idea because of the licensing.